misc/api-doc.js

```diff
@@ -13,21 +13,34 @@ const ANGULAR_LIFECYCLE_METHODS = [
   'ngAfterViewInit', 'ngAfterViewChecked', 'writeValue', 'registerOnChange', 'registerOnTouched', 'setDisabledState'
 ];
 
+function hasNoJSDoc(member) {
+  if (!member.symbol) {
+    return true;
+  }
+
+  const jsDoc = ts.displayPartsToString(member.symbol.getDocumentationComment());
+  return jsDoc.trim().length === 0;
+}
+
 function isInternalMember(member) {
   if (!member.symbol) {
     return true;
   }
 
   const jsDoc = ts.displayPartsToString(member.symbol.getDocumentationComment());
-  return jsDoc.trim().length === 0 || jsDoc.indexOf('@internal') > -1;
+  return jsDoc.indexOf('@internal') > -1;
 }
 
 function isAngularLifecycleHook(methodName) {
   return ANGULAR_LIFECYCLE_METHODS.indexOf(methodName) >= 0;
 }
 
+function isPrivate(member) {
+  return (member.flags & ts.NodeFlags.Private) !== 0;
```

Comment:
Those are bit-flags. Not sure if there is any documentation for this besides TS source code. But for the record: I'm not changing this code, just moving it around.